src/components/checkboxFilter/CheckboxGroupCollapsibleWithChildren.tsx

```diff
@@ -56,7 +56,7 @@ function CheckboxGroupCollapsibleWithChildren<
                 onToggleCheckboxGroup?.(item);
               }}
               w="full"
-              h="full"
+              h="sm"
```

Comment:
Are we fine reducing the clickable area for the height of the component to the height of the icon?

Reply:
yes
<img width="745" alt="Bildschirmfoto 2024-02-21 um 10 39 43" src="https://github.com/smg-automotive/components-pkg/assets/71456764/4b6361c3-a1b0-4a01-a69d-2b3d3e04bf5b">